packages/bot-web-ui/src/components/dashboard/joyrideConfig.tsx

```diff
@@ -0,0 +1,101 @@
+import React from 'react';
```

Comment:
the file name should be joyride-config.tsx as per our code convention

Reply:
done